po/tr.po

```diff
@@ -1513,26 +1497,20 @@ msgid "Remove"
 msgstr "Kaldır"
 
 #: src/gpodder/gtkui/desktop/episodeselector.py:138
-#, fuzzy
-#| msgid "Download"
 msgid "_Download"
-msgstr "İndir"
+msgstr "_İndir"
 
 #: src/gpodder/gtkui/desktop/episodeselector.py:282
 #: share/gpodder/ui/gtk/gpodderepisodeselector.ui.h:5
 #: share/gpodder/ui/gtk/gpodderpodcastdirectory.ui.h:4
-#, fuzzy
-#| msgid "Select all"
 msgid "Select _all"
-msgstr "Tümünü seç"
+msgstr "Tümünü _seç"
 
 #: src/gpodder/gtkui/desktop/episodeselector.py:286
 #: share/gpodder/ui/gtk/gpodderepisodeselector.ui.h:6
 #: share/gpodder/ui/gtk/gpodderpodcastdirectory.ui.h:5
-#, fuzzy
-#| msgid "Select none"
 msgid "Select _none"
-msgstr "Hiçbirini seçme"
+msgstr "Hiçbirini _seçme"
```

Comment:
> msgstr "Tümünü _seç"
> msgstr "Hiçbirini _seçme"

The letter after the underscore must be unique on the window where used. This would try to bind Alt+S to both actions. That dialog has `Select _all`, `Select _none`, `_Mark as old`, `_Cancel` and `_Download`. So `_Hiçbirini seçme` and `_Tümünü seç` would be better choices.

Reply:
Thank you for the warning. I corrected the relevant line **Seçimleri _kaldır**